### ISSUE TITLE
Mod Organizer no longer recongizing DLC

### ISSUE THREAD

**Issue by Wyverian** [TanninOne/modorganizer#975]
Hello, I recently cleaned my main and DLC .esms with FO4Edit and now suddenly MO doesn't recolonize I have the DLCs anymore. While this is most likely because I cleaned them and technically an issue with FO4Edit, I am more curious of how to make MO recognize them again. If you need any further detail please let me know. Thanks in advance.

**Comment by Hamguy21** [comment]
Bethesda released an update which requires ( * ) before every plugin and mod in your plugins.txt and DLCList.txt file. 
in AppData\Local\Fallout4 

I don't think MOD is updated yet to fix this issue and must be done manually.

**Comment by Wyverian** [comment]
Thank you for the quick reply I will double check the text file when I get back home. I however do doubt this was the issue unless this update happened right as I was cleaning the main and DLC esms, as MO was recognizing them prior to the cleaning.

**Comment by Wyverian** [comment]
Alright sorry for the double post but here is the update. I went through those two files you mentioned the plugins already had the * my DLClist did not so I added them. MO still does not recognize any DLC. 

Now I was able to actually test my game (I was going into work when I first posted this issue so didn't have the time). My game still does recognize that I have the DLCs, as I was apply to load in a save where I am in Nuka-World. I also loaded up another character who was in a settlement and created the armorsmith workbench and made some armor from the mod Armorsmith Extended (All DLC), despite MO giving me the "missing masters" warning.

So in reality the mods and the game see that I have the DLCs needed its just MO that can't seem to find the DLCs. I guess its not a huge issue because my game ignores MO's warnings and plays fine its just the warning symbols are unsightly. lol

**Comment by Gernash** [comment]
um, you have the "cleaned" files in MO2? if so you need to remove/rename(.old) them from the FO4 install directory, as MO2 it dosen't like having files of the same name in the root folder

**Comment by Wyverian** [comment]
Yes, I have FO4Edit added to my executable list in MO. Which root folder are you referring to exactly. I have the DLC files located in the primary Fallout 4 folder in steamapps like they always have been. I don't have any other DLC files anywhere else from what I can see.

Honestly I am unsure of what you mean outside of what I said above, I am not as adept as I like to seem XD sorry.

**Comment by Gernash** [comment]
OK.

If you have cleaned them like the S.T.E.P. guide and made folders and copied the DLC's into them and then cleaned them so you have both unmanaged and managed versions of the DLC's then you have to go into the FO4/data install directory and rename the files you have cleaned to "whatever.esm.old" as you can't have duplicate files in MO2 (files with the same name).

if you just cleaned them and it's not working then I'd just delete the cleaned files and repair the game in steam. I have not read anywhere you need to clean the FO4 files.

**Comment by Wyverian** [comment]
Alright uninstalling and reinstalling brought everything back.

**Comment by Hamguy21** [comment]
what i did to get my mods working was to just remove everything in the dlc and plugins list and let MO rebuild it using Sort plugins.
